src/app/modules/organization-management/components/organizations-table/organizations-table.component.html

```diff
@@ -71,12 +71,17 @@ <h3 id="all-organizations-header">{{ 'NAVIGATION.ALL_ORGANIZATIONS' | translate
       [attr.data-test]="'organizations__table__row-' + i"
     ></tr>
   </table>
-
-  <mat-paginator
-    *ngIf="dataSource.data.length > 10"
-    [pageSizeOptions]="[10, 20, 40]"
-    [pageSize]="pageSize"
-    (page)="pageSize = $event.pageSize"
-    showFirstLastButtons
-  ></mat-paginator>
+  <span
+    class="pagination-container"
+    [ngClass]="{
+      hidden: !showPagination
+    }"
+  >
+    <mat-paginator
```

Comment:
why do we need the span tag? can't we just apply the *ngIf directly to the mat0paginator tag?

Reply:
also, I think it is better to hide it with ngIf, rather than with a css display none. The ngIf takes the element completly out of the DOM and it is better

Reply:
It will not work with *ngIf, that's why the modification was made. First the number of elements is 0 and it is miscalculated on initialization.